FreeRTOS/Demo/ARC_NSIM_QEMU_GCC_MW/printf-stdarg.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2022 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+

```

Comment:
All files in current demo use Linux endline, but this file has Windows line endings.